zammad-ai-workflow/config.example.yaml

```
@@ -331,3 +331,15 @@ preparser:
 # API Configuration for REST endpoints
 api:
   api_key: null # Set in .env: ZAMMAD_AI_API__API_KEY
+
+# ============================================================================
+# Guardrail Service Configuration
+# ============================================================================
+guardrails:
```

YAMLlint (1.37.1)

[error] duplication of key "guardrails" in mapping

(key-duplicates)